src/index.js

```diff
@@ -154,7 +154,10 @@ export default function(babel) {
       },
       ImportDeclaration(path) {
         const source = path.node.source;
-        if (t.isStringLiteral(source) && /^jsbi$/i.test(source.value)) {
+        if (t.isStringLiteral(source) &&
+            // Match exact "jsbi" or ".../jsbi.mjs" paths.
+            (/^jsbi$/i.test(source.value) ||
+             /[/\\]jsbi.mjs$/i.test(source.value))) {
```

Comment:
Let's escape the `.` here, since it has special meaning within RegExp patterns.

```suggestion
             /[/\\]jsbi\.mjs$/i.test(source.value))) {
```